### PR TITLE
Hardcode the canton version in BuildInfo for now

### PR DESCRIPTION
**Pull request by paulbrauner-da**
Until https://github.com/DACH-NY/canton/issues/15106 is resolved, hardcode the canton version in the BuildInfo synthetized by our genrule when building canton with bazel. We currently set it to `0.0.0` in our development environment, which is not a valid canton version.

### CODE REVIEW

File: canton/BUILD.bazel [hunk 1]
```diff
@@ -23,7 +25,7 @@ cat << EOF > $@
 package com.digitalasset.canton.buildinfo
 
 case object BuildInfo {{
-  val version: String = "{sdk_version}"
+  val version: String = "2.7.0-SNAPSHOT"
```
Comment (remyhaemmerle-da) [inline canton/BUILD.bazel:28]:
```suggestion
  val version: String = "2.8.0-SNAPSHOT"
```

File: canton/BUILD.bazel [hunk 2]
```diff
@@ -23,7 +25,7 @@ cat << EOF > $@
...
   val scalaVersion: String = "{scala_version}"
   val sbtVersion: String = "bazel"
   val damlLibrariesVersion: String = "2.7.0-SNAPSHOT"
```
Comment (rgugliel-da) [inline canton/BUILD.bazel:31]: Here as well ? (2.8)

File: canton/BUILD.bazel (hunk 1 above)
Comment (paulbrauner-da) [reply canton/BUILD.bazel:28]: Done

File: canton/BUILD.bazel (hunk 2 above)
Comment (paulbrauner-da) [reply canton/BUILD.bazel]: Actually used {sdk_version} as that's the version we're compiling with.

**Review by remyhaemmerle-da — APPROVED** [review]
:+1:

**Pull request merged**